apps/studio/pages/project/[ref]/index.tsx

```diff
@@ -70,9 +70,9 @@ const Home: NextPageWithLayout = () => {
     projectRef: project?.ref,
   })
 
-  const tablesCount = tablesData?.length ?? 0
-  const functionsCount = functionsData?.length ?? 0
-  const replicasCount = (replicasData?.length ?? 1) - 1
```

Comment:
hmm iirc we need to minus one here, cause replicasData also includes the primary database (which shouldn't be considered as a replica) - i'll double check on this branch